### PR TITLE
Updates packages, including package:multicast_dns.

### PR DESCRIPTION
**Pull request by zanderso**
## Description

This is a run of `flutter update-packages --force-upgrade` to pick up bug fixes to `package:multicast_dns`.

## Related Issues

https://github.com/flutter/flutter/issues/41085

## Tests

Added a test of parsing the authCode out of the mdns query response, which is related to the bug fixes in `package:multicast_dns`.

## Breaking Change

Does your PR require Flutter developers to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (Please read [Handling breaking changes]). *Replace this with a link to the e-mail where you asked for input on this proposed change.*
- [x] No, this is *not* a breaking change.

### CODE REVIEW

**Comment by zanderso** [comment]
Added a test. PTAL.

File: packages/flutter_tools/pubspec.yaml [hunk 1]
```diff
@@ -11,17 +11,17 @@ dependencies:
   # To update these, use "flutter update-packages --force-upgrade".
   archive: 2.0.10
   args: 1.5.2
-  dwds: 0.6.2
+  dwds: 0.7.1
```
Comment (jonahwilliams) [inline packages/flutter_tools/pubspec.yaml]: Did you need to pull in the change from https://github.com/flutter/flutter/pull/41805/files#diff-ce38e968fc555ba18c509b8fac294f0aR266 for this

Comment (zanderso) [reply packages/flutter_tools/pubspec.yaml]: Done.

**Review by jonahwilliams — APPROVED** [review]
LGTM for web changes. @christopherfujino for mdns changes

**Review by christopherfujino — APPROVED** [review]
This lgtm, I would just recommend adding some mention of the mDNS change to the commit message in case that change breaks something.

**Comment by zanderso** [comment]
> This lgtm, I would just recommend adding some mention of the mDNS change to the commit message in case that change breaks something.

Done.

**Comment by zanderso** [comment]
The tree is really green. Landing.

**Pull request merged**